src/components/link/index.tsx

```diff
@@ -39,10 +39,10 @@ const Link = ({ children, href, opensInNewTab, ...restProps }: LinkProps) => {
 	 */
 	let ariaDescribedBy: LinkProps['aria-describedby']
 	if (shouldRenderScreenReaderOnlyMessage) {
-		ariaDescribedBy += opensInNewTabLabelId
+		ariaDescribedBy = opensInNewTabLabelId
 	}
 	if (restProps['aria-describedby']?.length > 0) {
-		ariaDescribedBy += ` ${restProps['aria-describedby']}`
+		ariaDescribedBy = ` ${restProps['aria-describedby']}`
 	}
```

Comment:
Is there ever a case where both `shouldRenderScreenReaderOnlyMessage` and `restProps['aria-describedby']?.length > 0` are true? If so, would we want to assign `ariaDescribedBy` as `opensInNewTabLabelId +  ${restProps['aria-describedby']}`. The way the code is currently leads me to believe that. Otherwise, why the addition assignment and the leading space character?

Reply:
ohhh good point. I someone read that above line as an `else if` 🤦🏻 . I'll change the code to handle this case.